### DISCUSSION TITLE
Dynamic commands only work on file creation. Not inserted into an existing file.

### DISCUSSION THREAD

**Discussion by AlexRimor** [SilentVoid13/Templater#530] [Help]
So these commands work (converting to the relevant dates) when I am creating a file from a template.
However when I am inserting them into an _existing_ file they just show the code.
This happens even if I change to preview mode, and close and re-open the file.  The templates are both in the same folder.

Any suggestions about what I'm doing wrong?

```
creation date: <%+ tp.file.creation_date() %>
modification date: <%+ tp.file.last_modified_date("dddd Do MMMM YYYY HH:mm:ss") %>
```

**Comment by welpdx** [comment]
Since these are dynamic commands, previewing it should show the dates. You can still see the code when you go in edit mode though. 

There used be some issues with dynamic commands (#188, #190, #195, #204) but they should be [fixed](https://github.com/SilentVoid13/Templater/blob/master/CHANGELOG.md#171-2021-05-23) now.

Do you still have this problem where preview still shows the dynamic code?
